shesmu-server/src/test/resources/run/group-synthetic.shesmu

```diff
@@ -0,0 +1,5 @@
+Input test;
```

Comment:
Both of these test files have `synthetic` in the name but I don't actually think that's an issue.